README.md

```diff
@@ -34,16 +34,130 @@ that the operator is able to understand and process.
 * [git][git_tool]
 * [go] version 1.12.5+
 * [kubernetes] version v1.11.0+
-* [oc] version v3.11+
+* [kubectl] version v1.11+
 * Access to a Openshift v3.11.0+ cluster.
 * A user with administrative privileges in the OpenShift cluster.
 
 ## Getting started
 
-WORK IN PROGRESS
+### Deploy Apicast gateway providing configuration secret
+
+This is a very basic example to deploy apicast with a very simple configuration.
+The [json configuration file](https://github.com/3scale/APIcast/blob/master/examples/configuration/echo.json)
+will make apicast behave like an echo api endpoint.
+See more [examples of configuration files](https://github.com/3scale/APIcast/tree/master/examples/configuration).
+
+Create secret with the configuration file:
+
+```sh
+curl https://raw.githubusercontent.com/3scale/APIcast/master/examples/configuration/echo.json -o $PWD/config.json
+kubectl create secret generic apicast-echo-api-conf-secret --from-file=$PWD/config.json
+```
+
+Note that config file must be called `config.json`.
+This is [Apicast CRD reference](doc/apicast-crd-reference.md) requirement
+
+Create [apicast custom resource](doc/apicast-crd-reference.md):
+
+```yaml
+$ cat my-echo-apicast.yaml
+apiVersion: apps.3scale.net/v1alpha1
+kind: APIcast
+metadata:
+  name: my-echo-apicast
+spec:
+  exposedHost:
```

Comment:
Maybe we shouldn't specify exposedHost in the basic example?
It's not really necessary (only if you want it to be externally accessible) and depending on the situation (if they don't have a default ingress controller set) they should create an Ingress Controller so it might not be ideal for a "basic" scenario

Reply:
For me it does not make sense deploying apicast without ingress resource, i.e. not being exposed externally. Does it? This makes me think that maybe CRD should **require** exposedHost (I would call it `domainName`). WDYT?

Which ingress controller they use does not matter... this is another topic. Am I right?

Reply:
To me it makes sense.

You could be using apicast and connecting it with other elements in the Kubernetes cluster.

I don't think we should require exposedHost at all

Reply:
It seems we cannot predict what the use cases are going to be. We will leave `exposedHost`  as optional. Still valid the proposal of changing name to `domainName`.

However, I would leave the example as it is because it is a "getting started" basic example and you probably want to curl it from your terminal, hence needs to be exposed.

Reply:
You don't really need to expose to curl from the terminal. You can proxy the pod / service through kubectl port-forward. It might be easier to just expose the host so we can leave it that way if you like it, although the user will have to have to set a resolvable DNS name in that case.

The reason I chose exposedHost (and host as the subfield) and not domain is due to I identify "domain" with the name given to the "network domain" and host to the specific host in that domain.